src/vs/workbench/contrib/tasks/browser/abstractTaskService.ts

```diff
@@ -258,7 +267,8 @@ export abstract class AbstractTaskService extends Disposable implements ITaskSer
 		@ITextModelService private readonly textModelResolverService: ITextModelService,
 		@IPreferencesService private readonly preferencesService: IPreferencesService,
 		@IViewDescriptorService private readonly viewDescriptorService: IViewDescriptorService,
-		@ILogService private readonly logService: ILogService
+		@ILogService private readonly logService: ILogService,
+		@ISqlTaskService private readonly sqlTaskService: ISqlTaskService
```

Comment:
so now we can have one task tie in its log/state in activities pane as well as output pane - or it will have to pick one or another?

Reply:
Yeah - how does this fit into the VS Code tasks system? Are we planning on making the existing ADS tasks more like VS Code tasks in how they're defined and ran?

Reply:
The way ADS task works right now will work the way it does. What this will do is introduce the VS Code Task system functionality, and those tasks will also be shown in the ADS Tasks Panel. I didn't want to change the ADS Task system since there are extensions that depend on it.